model/riscv_insts_zicsr.sail

```diff
@@ -241,6 +250,15 @@ function writeCSR (csr : csreg, value : xlenbits) -> unit = {
     /* user mode: seed (entropy source). writes are ignored */
     (0x015,  _) => write_seed_csr(),
 
+    /* vector csr */
+    (0x008, _) => { vstart = sail_zero_extend(value[(vstart_length - 1) .. 0], 9); Some(EXTZ(vstart)) },
```

Comment:
Use EXTZ

Reply:
The EXTZ issues in the CSR functions are fixed.